core/sdk/src/api/manifest/device_manifest.rs

```diff
@@ -346,38 +342,32 @@ pub enum AppLauncherMode {
     Internal,
 }
 
-fn model_friendly_names_default() -> HashMap<String, String> {
-    HashMap::default()
-}
-
-fn settings_defaults_per_app_default() -> HashMap<String, SettingsDefaults> {
-    HashMap::default()
-}
-
-fn default_values_default() -> DefaultValues {
-    DefaultValues {
-        country_code: "US".to_string(),
-        language: "en".to_string(),
-        locale: "en-US".to_string(),
-        name: "Living Room".to_string(),
-        captions: captions_default(),
-        voice: voice_guidance_default(),
-        additional_info: additional_info_default(),
-        allow_acr_collection: false,
-        allow_app_content_ad_targeting: false,
-        allow_business_analytics: default_business_analytics(),
-        allow_camera_analytics: false,
-        allow_personalization: false,
-        allow_primary_browse_ad_targeting: false,
-        allow_primary_content_ad_targeting: false,
-        allow_product_analytics: false,
-        allow_remote_diagnostics: false,
-        allow_resume_points: false,
-        allow_unentitled_personalization: false,
-        allow_unentitled_resume_points: false,
-        allow_watch_history: false,
-        skip_restriction: "none".to_string(),
-        video_dimensions: default_video_dimensions(),
+impl Default for DefaultValues {
+    fn default() -> Self {
```

Comment:
This is all the same values, just put in a Default impl so it can be called from unit test instead of only when the manifest file is read.